src/lib/Libwin/env.c

```diff
@@ -48,9 +48,67 @@ static char	sysrootdir[MAXPATHLEN+1] = "";
 static char	sysdrive[MAXPATHLEN+1] = "";
 static char	temp_path[MAXPATHLEN+1] = "";
 static char	user_domain[PBS_MAXHOSTNAME+1] = "";
+static char	value_buf[ENV_BUF_SIZE] = "";
 /**
  * @file	env.c
  */
+/**
+ * @brief
+ * 	_setenv_win: Set an environment variable
+ *  
+ * @param[in] key - string holding key for environment variable
+ * @param[in] value - string holding value for environment variable
+ * @param[in] interrupt - int value to override existing value, not used in Windows
+ *	
+ * @return	int
+ * @retval	0			success
+ * @retval	non-zero	error
+
+ */
+int
+_setenv_win(char* key, char* value, int overwrite)
+{
+	errno = 0;
+
+	if(!GetEnvironmentVariable(key, value_buf, ENV_BUF_SIZE) & !overwrite)
```

Comment:
I think this should be &&

Reply:
I addressed this comment and changed it to &&. But I did a commit --amend, so it didn't show up as a new commit, sorry about that!